examples/Demo/Shared/wwwroot/docs/DesignTokens.md

```diff
@@ -337,3 +337,14 @@ To make this work, a link needs to be created between the Design Token component
 ## Colors for integration with specific Microsoft products
 If you are configuring the components for integration into a specific Microsoft product, the following table provides `AccentBaseColor` values you can use. 
 *The library offers an `OfficeColor` enumeration which contains the specific accent colors for 17 different Office applications.*
+
```

Comment:
The table was removed because it wasn't complete and I thought it wasn't really helpful to show the hex codes.
If we put it back in, it must contain all enum values.
But I rather have an image which showes the actual colors along with the names.

Reply:
Just found [OfficeColors](https://github.com/microsoft/fluentui-blazor/blob/dev/src/Core/Enums/OfficeColor.cs) 😄.

> But I rather have an image which showes the actual colors along with the names

I can mock something up for you to take a look at.

Reply:
@vnbaaij 

In order to have a table with colors without inserting an image was to do it in code. I created a new component `OfficeColorTable` and placed it following the `MarkdownSection`. I also tested with both Light and Dark themes.

There is a funny glitch where the table appears before the markdown section is rendered. The solution would be delay rendering the table till after the markdown has been rendered. I'm looking into that next.

How does this look to you?

Reply:
I think this is fine. Yes, it does show the table first. but I find that not that big of a deal.

There is a component in the datagrid infrastructure `Defer` that moves thing to the end of the renderqueue. I tried that locally but it did not change the render order. ou could look into that code but only if it bugs you so much that you have to change it. Agian, current solution is fine by me.

Reply:
I have another idea on how to solve this perhaps... Let me try some things.